setup.cfg

```diff
@@ -1,5 +1,24 @@
+[metadata]
+name = Tubes
+version = 0.2.0
+description = Flow control and backpressure for event-driven applications.
+license = MIT
+url = https://github.com/twisted/tubes/
+
+[options]
+packages = find:
+package_dir = tubes=tubes
+install_requires =
+    six
+    Twisted
+include_package_data = True
+
+[options.packages.find]
+exclude =
```

Comment:
I'm pretty sure an empty exclude is the default
```suggestion
```

Reply:
indeed, so removed

9:15 AM @graingert: and `[options.packages.find] exclude` defaults to empty?
9:17 AM @uranusjr: yes